CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing to Terramate
+
+## A Word Before We Begin
+
+First and foremost, we'd like to express our gratitude to you for taking the time to contribute. We don't accept feature contributions by the community as we are currently sticking to an internal roadmap. This may change in the future.
+
+We welcome and appreciate all bug fix contributions via [Pull Requests](https://github.com/mineiros-io/terramate/pulls) along the [GitHub Flow](https://guides.github.com/introduction/flow/).
+
+Thanks!
+
+## Contribution Workflow
+
+For bug reports or requests, please submit your issue in the appropriate repository.
+
+We advise that you open an issue and ask the [CODEOWNERS](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners) and community prior to starting a contribution. This is your chance to ask questions and receive feedback before writing (potentially wrong) code. We value the direct contact with our community a lot, so don't hesitate to ask any questions.
+
+For contributing to Terramate, please follow these steps:
+
+1. Within your fork of
+   [Terramate](https://github.com/mineiros-io/terramate), create a
+   branch for your contribution. Use a meaningful name.
+1. Create your contribution, meeting all
```

Comment:
```suggestion
2. Create your contribution, meeting all
```

Reply:
that change is really not necessary - markdown is creating the continues list for us :) but ok for readability

Reply:
we should not block the PR because of something like this tho I assume